CMakeLists.txt

```diff
@@ -331,7 +330,7 @@ if(CMEPS)
 endif()
 
 if(CDEPS)
-  message("Do CDEPS things here")
+  add_dependencies(ufs cdeps::datm)
```

Comment:
I think this needs to be revisited again because we have also other data components such as docn. Adding list of dependencies under CDEPS fi statement has no any meaning. I think in addition to the CDEPS flag in here we could also need DATM, DOCN etc. in here. So, this will give a capability to us to turn on/off particular data component. I could fine tune the build in the HAFS level to support other data components but maybe you might want to do it in this PR.

Reply:
Agreed. 
This is a placeholder for current data atmosphere model and an example on how to link with it. 
When other data components from CDEPS are used in well defined APP handles, this logic will need to be updated/replaced accordingly. 

So far, there is only one data component in the UFS-weather-model and that is the data atmosphere. 

- I don't think that needs to be figured out in this PR that simply adds a build of CDEPS as a component of the UFS weather model.

Reply:
@aerorahul okay. that is fine. I could make the required changes in the HAFS app and this will eventually go back to UFS model.

Reply:
Definitely. 
When this comes back from HAFS, we can have a discussion then.